src/core/config/npo/snapshots/tailcall__core__config__npo__tracker__tests__cycles_with_resolvers.snap

```diff
@@ -2,5 +2,4 @@
 source: src/core/config/npo/tracker.rs
 expression: actual
 ---
-query { f1 { f1 { f2 } } }
 query { f1 { f2 } }
```

Comment:
here one path is dropped, but it was basically the same path so if batching is added it'd resolve both branches. I don't think there is difference if we show both